packages/mobile/src/screens/chat-screen/ChatScreen.tsx

```diff
@@ -182,10 +193,11 @@ export const ChatScreen = () => {
   )
   const unreadCount = chat?.unread_message_count ?? 0
   const isLoading =
-    chat?.messagesStatus === Status.LOADING && chatMessages?.length === 0
+    (chat?.messagesStatus ?? Status.LOADING) === Status.LOADING &&
```

Comment:
Probably should put this logic in its own selector but let's merge this for QA for now.